tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1146,4 +1146,59 @@ public function testDoesNotListIndexesImplicitlyCreatedByForeignKeys()
         $this->assertArrayHasKey('explicit_fk1_idx', $indexes);
         $this->assertArrayHasKey('idx_3d6c147fdc58d6c', $indexes);
     }
+
+    /**
+     * @dataProvider commentsProvider
+     *
+     * @param string $comment
+     * @param string $expected
+     * @param string $currentType
+     * @return void
+     */
+    public function testExtractDoctrineTypeFromComment($comment, $expected, $currentType) : void
+    {
+        $result = $this->_sm->extractDoctrineTypeFromComment($comment, $currentType);
+        self::assertSame($expected, $result);
+    }
+
+    /**
+     * @return array
+     */
+    public function commentsProvider() : array
+    {
+        $currentType = 'current type';
+
+        return array(
```

Comment:
Please use short-array syntax

Reply:
Please define the indexes of the array with a short description of what the set of parameters is about, in order to improve PHPUnit's messages. Like:

```php
return [
    'invalid custom type comments' => ['should.return.current.type', $currentType, $currentType],
    'valid custom type with dots'  => ['(DC2Type:type.should.return)', 'type.should.return', $currentType],
];
```